client/views/actions/whowas.tpl

```diff
@@ -0,0 +1,10 @@
+<p>WHOWAS info for {{> ../user_name nick=whowas.nick}}</p>
+<dl class="whois">
+
+	<dt>Host mask:</dt>
+	<dd class="hostmask">{{whowas.ident}}@{{whowas.host}}</dd>
+
+	<dt>Real name:</dt>
+	<dd>{{{parse whowas.real_name}}}</dd>
```

Comment:
Dynamically looping over all whowas.* fields may bring forward compatibility with changes to kiwiirc/irc-framework's event emission, as long as we are OK with showing everything that a normal WHOWAS would also show and don't want to exclude something for convenience.